extensions/wikia/AdEngine/js/config/desktop.js

```diff
@@ -67,13 +67,21 @@ define('ext.wikia.adEngine.config.desktop', [
 			return [];
 		}
 
+		// Force Turtle
+		if (context.forcedProvider === 'turtle') {
+			log(['getProvider', slotName, 'Turtle (wgAdDriverForcedProvider)'], 'info', logGroup);
+			return [adProviderTurtle];
+		}
+
 		// Force OpenX
-		if (context.forceProviders.openX) {
-			log(['getProvider', slotName, 'OpenX (wgAdDriverForceOpenXAd)'], 'info', logGroup);
+		if (context.forcedProvider === 'openx') {
+			log(['getProvider', slotName, 'OpenX (wgAdDriverForcedProvider)'], 'info', logGroup);
 			return [adProviderOpenX];
 		}
 
-		if (context.forceProviders.liftium) {
+		// Force Liftium
+		if (context.forcedProvider === 'liftium') {
+			log(['getProvider', slotName, 'Liftium (wgAdDriverForcedProvider)'], 'info', logGroup);
 			return [adProviderLiftium];
 		}
```

Comment:
I wonder if we could force one of all providers in `for` loop instead of multiple `if` statements, then it would be easier to test another providers also, but it was never needed so it's just an idea.